### DISCUSSION TITLE
There should be a way to disable "Edit account" link

### DISCUSSION THREAD

**Discussion by Gummibeer** [filamentphp/filament#392] [Ideas]
The user dropdown contains an "Edit account" link which should be optional as not all apps allow to customize the account/profile.

There should be any kind of `if` around that block in `resources/views/components/layouts/app.blade.php`.
```html
<x-filament::dropdown-link href="{{ route('filament.account') }}">
    {{ __('filament::nav.dropdown.account.label') }}
</x-filament::dropdown-link>
```

**Comment by danharrin** [comment]
Feel free to open a PR that adds a config option for this, along with an authorisation check not the edit account page to match.
